src/OMSimulatorLib/Model.cpp

```diff
@@ -351,6 +351,7 @@ oms_status_enu_t oms::Model::exportToSSD(pugi::xml_node& node, pugi::xml_node& s
   node.append_attribute("xmlns:ssv") = "http://ssp-standard.org/SSP1/SystemStructureParameterValues";
   node.append_attribute("xmlns:ssm") = "http://ssp-standard.org/SSP1/SystemStructureParameterMapping";
   node.append_attribute("xmlns:ssb") = "http://ssp-standard.org/SSP1/SystemStructureSignalDictionary";
+  node.append_attribute("xmlns:oms") = "https://raw.githubusercontent.com/OpenModelica/OMSimulator/schema/oms.xsd";
```

Comment:
I guess it should be `https://raw.githubusercontent.com/OpenModelica/OMSimulator/master/schema/oms.xsd`

Reply:
changed the url see https://github.com/OpenModelica/OMSimulator/pull/761/commits/e1d756184319f9c850fb05ffdb95af080212feef